build/teamcity-test.sh

```diff
@@ -16,9 +24,6 @@ run_cockroach() {
   # Start a CockroachDB server, wait for it to become ready, and arrange
   # for it to be force-killed when the script exits.
   rm -f "$urlfile"
-  # Clean out a past CockroachDB instance. This will clean out leftovers
-  # from the build agent, and also between CockroachDB runs.
-  cockroach quit --insecure || true
```

Comment:
`cockroach quit` no longer exists

Reply:
do we need something else that shuts down the process?